exporter/awskinesisexporter/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awskinesisexporter
 
-go 1.22.0
+go 1.22.7
+
+toolchain go1.22.8
```

Comment:
I suppose we don't want the `toolchain` change?

Reply:
@songy23 we are reviewing and will update shortly